src/Util/Lock/MemcacheLockDriver.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Friendica\Util\Lock;
+
+use Friendica\Core\Cache;
+use dba;
+
+class MemcacheLockDriver implements ILockDriver
```

Comment:
This name is misleading, the `Cache` class doesn't necessarily use Memcache. Please rename to `CacheLockDriver`.